### ISSUE TITLE
Acessing Files or URLs Tab result in an Error 500 as Anonymous User with Signups disabled

### ISSUE THREAD

**Issue by richardmeilinger** [pglombardo/PasswordPusher#716]
## 🐛 Bug Report

Accessing Files or URLs tab result in an Error 500 as Anonymous User with signups disabled.

## 🔬 How To Reproduce

1. set env to `PWP__DISABLE_SIGNUPS=true`
2. try to acces files or urls tab

### Environment

- [x] Docker
  - [x] pwpush-postgres

If applicable, what version of Password Pusher?

latest

### Screenshots

![image](https://user-images.githubusercontent.com/36570827/213887062-f429964a-67f8-40c7-a518-2fc2a5136b91.png)
![image](https://user-images.githubusercontent.com/36570827/213886972-48f20b5d-0ce4-450b-9426-405912d97a71.png)

## 📈 Expected behavior

show message or disable tabs for anonymous users

## 📎 Logs

```
F, [2023-01-21T20:54:34.573222 #62] FATAL -- : [4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c] ActionView::Template::Error (undefined local variable or method `new_user_registration_path' for #<ActionView::Base:0x0000000006b3c8>):
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     15:             <p class="card-text">
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     16:             </p>
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     17:             <a href="<%= new_user_session_path %>" class="btn btn-primary">Log In</a>
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     18:             <a href="<%= new_user_registration_path %>" class="btn btn-success">Sign Up</a>
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     19:         </div>
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     20:     </div>
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]     21: </div>
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c]
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c] app/views/file_pushes/new_anonymous.html.erb:18
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c] app/controllers/file_pushes_controller.rb:81:in `block (2 levels) in new'
[4af30a54-5fd0-40b6-a2c0-7ce5e583f63c] app/controllers/file_pushes_controller.rb:80:in `new'
```

**Comment by pglombardo** [comment]
Hi @richardmeilinger - good find.  I'll put in a fix for this soon.  Thanks for reporting

**Comment by pglombardo** [comment]
Hi @richardmeilinger - the fix for this is building in [1.24.7](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.24.7) now.  Thanks for pointing this out!

**Comment by richardmeilinger** [comment]
hi @pglombardo - thanks for the fix.
